docs/features/models.md

```diff
@@ -29,14 +29,13 @@ on your model to disable this behavior.
 
 ### Automatic object ownership
 
-If you want to keep track of object ownership automatically, you can use the `CurrentUserMiddleware`.
-Just make sure, you'll insert it **after** djangos `AuthenticationMiddleware`:
+If you want to keep track of object ownership automatically, you can use the `CurrentRequestMiddleware`:
```

Comment:
comment about async-safety

Reply:
Treated in 2fe2cb3e60edeb9dedac757ccbf7209b1228669e

Although we now _do_ state that the middleware can be used in an async context, which we didn't do before.

I would however expect no issues from using it in an async context, assuming Django's auto sync/async transitioning works as documented (see [Django middleware - async support](https://docs.djangoproject.com/en/3.1/topics/http/middleware/#asynchronous-support))